src/devices/UFireIse/README.md

```diff
@@ -0,0 +1,77 @@
+﻿# μFire ISE Probe Interface - I2C Driver
+
+## Summary
+
+The μFire ISE Probe Interface is an I²C sensor that can read a pH probe. Attach a waterproof temperature sensor for temperature compensation with the attached connector.
+
+## Usage
+
+You can find an example in the [sample](./samples/uFireIse.Sample.cs) directory. 
+
+
+### Basic
+It is possible to read the basic value (Electric Potential) from the probe.
+   
+```csharp
+using (UFireIse uFireIse = new UFireIse(device))
+{
+    Console.WriteLine("mV:" + uFireIse.Measure().Millivolts);
+}
+```
+
+### Orp 
+To read the ORP (OxidationReducationPotential) value use this example
+   
+```csharp
+using (UFireOrp uFireOrp = new UFireOrp(device))
+{
+	if (uFireOrp.TryMeasureOxidationReducationPotential(out ElectricPotential orp))
+	{
+	    Console.WriteLine("Eh:" + orp.Millivolts);
+	}
+	else
+	{
+		Console.WriteLine("Not possible to measure pH");
+	}
+}
+```
+
+### Calibration 
+Calibration of the probe using a single solution. 
+Put the probe in a solution where the pH (Power of Hydrogen) value is known (in this example we assume it is 7).
+   
+```csharp
+ using (UFirePh uFire_pH = new UFirePh(device))
+{
+    uFire_pH.CalibrateSingle(7);
```

Comment:
really nice, might also be worth adding if this calibration stays if you reboot device/app or you need to re-calibrate every time